### DISCUSSION TITLE
Should we remove `http` as a dependency from `update_available_ios`?

### DISCUSSION THREAD

**Discussion by mateusfccp** [mateusfccp/update_available#31] [Polls]
The less dependencies a package have, the better. `dart:io` already provides `HttpClient`, and as we are using `http` only for a single request on `update_available_ios`, I am considering removing it in favor of `HttpClient`.

I don't know any disadvantage of using `HttpClient` except for slightly more complexity. Considering that we use it a single time, however, it should be negligible.

WDYT?

**Comment by mateusfccp** [comment]
@kuhnroyal

**Comment by kuhnroyal** [comment]
Since this is an iOS only package, it is completely fine to drop `http`. The advantage of `http` is that it also works in web but this is not required.

**Comment by mateusfccp** [comment]
This change has been introduced in daf4288, and published in `3.0.0-dev.2`.
